hydroengine_service/datasets/dataset_visualization_parameters.json

```diff
@@ -171,5 +171,52 @@
 			"50th": ["#042333", "#3c0912", "#730e27", "#a62225", "#c0583b", "#d08b73", "#dfbcb0", "#f1edec"],
 			"90th": ["#042333", "#3c0912", "#730e27", "#a62225", "#c0583b", "#d08b73", "#dfbcb0", "#f1edec"]
 		}
+	},
+    "projects/dgds-gee/chasm/waves": {
+		"type": "ImageCollection",
+		"bandNames": {
+			"significant_wave_height": "significant_wave_height",
+			"wave_direction": "wave_direction",
+            "wave_period": "wave_period",
+            "turbulence": "turbulence"
+		},
+		"min": {
+			"significant_wave_height": 1.0,
+			"wave_direction": 0.0,
+            "wave_period": 5.0,
+            "turbulence": 0.0
+		},
+		"max": {
+			"significant_wave_height": 3.0,
+			"wave_direction": 360.0,
+            "wave_period": 9.0,
+            "turbulence": 0.5
+		},
+		"palette": {
+			"significant_wave_height": ["#042333", "#3c0912", "#730e27", "#a62225", "#c0583b", "#d08b73", "#dfbcb0", "#f1edec"],
```

Comment:
from cmocean?

Reply:
Yes, and same palette we use for other wave data.